doc/apps/dgst.pod

```diff
@@ -14,21 +14,24 @@ B<openssl> B<dgst>
 [B<-binary>]
 [B<-r>]
 [B<-non-fips-allow>]
-[B<-out filename>]
-[B<-sign filename>]
-[B<-keyform arg>]
-[B<-passin arg>]
-[B<-verify filename>]
-[B<-prverify filename>]
-[B<-signature filename>]
-[B<-hmac key>]
+[B<-out> I<filename>]
+[B<-sign> I<filename>]
+[B<-keyform> I<arg>]
+[B<-passin> I<arg>]
+[B<-verify> I<filename>]
+[B<-prverify> I<filename>]
+[B<-signature> I<filename>]
+[B<-hmac> I<key>]
+[B<-mac> I<alg>]
+[B<-macopt> I<nm>B<:>I<v> ...]
+[B<-rand> I<files>]
 [B<-non-fips-allow>]
 [B<-fips-fingerprint>]
-[B<file...>]
+[I<file ...>]
```

Comment:
Okay, I pushed a commit that de-italicizes all the ellipses in the reformatted files. What do you think?

<img width="699" alt="screen shot 2015-09-18 at 2 05 32 am" src="https://cloud.githubusercontent.com/assets/2618447/9952942/e53a2612-5da9-11e5-8d74-206e45544016.png">